default/serializers.py

```
@@ -0,0 +1,68 @@
+        model = Comment
+        fields = ['id', 'content', 'author', 'created_at']
+        read_only_fields = ['created_at']
+
+
+class PostListSerializer(serializers.ModelSerializer):
+    author = UserSerializer(read_only=True)
+    comments_count = serializers.SerializerMethodField()
+    
+    class Meta:
+        model = Post
+        fields = ['id', 'title', 'author', 'created_at', 'updated_at', 'comments_count']
```

Ruff (0.12.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)